cmd/ocsp-updater/main.go

```diff
@@ -268,6 +273,73 @@ func (updater *OCSPUpdater) oldOCSPResponsesTick(batchSize int) {
 	updater.generateOCSPResponses(statuses)
 }
 
+func (updater *OCSPUpdater) getSerialsIssuedSince(since time.Time, batchSize int) ([]string, error) {
```

Comment:
Sorry I didn't comment on this comment earlier. Are you sure that's what we want? That makes it impossible to use batchSize as a means to control the rate of signing we request. If we have a large backlog, OCSP Updater will be pushing signings through as fast as it can go, rather than at a specific chosen rate.

Maybe I'm missing something and your plan is to address this with RPC backpressure instead?